docs/modules/SDDs/pages/0022-managed-services-in-cluster.adoc

```diff
@@ -0,0 +1,166 @@
+= SDD 0022 - Managed Services in Cluster
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:
+:sdd_reviewers:
+:sdd_date:      2020-06-19
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+Besides the system services Project Syn also provides services to the user of the platform. This SDD describes how some kind of managed services on Kubernetes can be achieved with Project Syn. It's considered to be the initial version of this concept.
+====
+
+== Motivation
+
+While the current state of Project Syn focuses on delivering system services to Kubernetes clusters, it should also take application services into consideration. There needs to be a way to install, configure and maintain application services and make them available to the user of the platform.
+
+As application services are considered services like databases, caches, queues and other similar applications (sometimes called "Middleware"). They're usually used by the end user application running on a Kubernetes cluster, but could also be used by applications running outside of the cluster (f.e. in another cluster or even a VM).
```

Comment:
"f.e." is not a thing in English. We should use the established "e.g." when shortening "for example"